puma/plot_base.py

```diff
@@ -431,8 +431,8 @@ def set_y_lim(self):
         for i, ratio_axis in enumerate(self.ratio_axes):
             if self.ymin_ratio[i] or self.ymax_ratio[i]:
```

Comment:
Thanks for the bugfix!
Can you please also change this line?
Otherwise, it won't affect the situation when `self.ymin_ratio[i] =0` and `self.ymax_ratio[i] = None`

Reply:
Sure! Sorry for not spotting it earlier!